util/cron/common-asan.bash

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+#
+# Configure environment for testing AddressSanitizer.
+
+CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+
+export CHPL_MEM=cstdlib
+export CHPL_TASKS=fifo
+export CHPL_SANITIZE_EXE=address
```

Comment:
Any reason to not sanitize the compiler too (`export CHPL_SANITIZE=address`)?

Reply:
I just remembered that it was causing compiler performance slowdowns, but maybe I was wrong?

Reply:
The early implementation you were using was throwing `-g -O1`, which I think was the main cause of overhead. I'd probably start with compiler sanitizing enabled, and we can evaluate if it's too much slowdown later.

Reply:
Ah, right. Sounds good.